drivers/hmis_external_apis/app/views/layouts/hmis_external_apis/external_forms.haml

```diff
@@ -72,6 +72,9 @@
       fieldset > .form-group:has(> .form-check) {
           margin-bottom: 0px;
       }
+      .hide {
```

Comment:
We import the bootstrap css, which should have a class for this. Did you try `hidden` or `d-none`?

Reply:
It seems like d-none works but sets `!important`, preventing the jquery `.hide()`/`.show()` calls from functioning correctly.